### ISSUE TITLE
Icons are not visible with dark themes

### ISSUE THREAD

**Issue by medmedin2014** [veusz/veusz#491]
When I use dark theme, the icons are really hard to see and differentiate

```
Operating System: Manjaro Linux
KDE Plasma Version: 5.20.4
KDE Frameworks Version: 5.76.0
Qt Version: 5.15.2
Kernel Version: 5.4.80-2-MANJARO
OS Type: 64-bit
```

![path843](https://user-images.githubusercontent.com/7027319/102927874-156b3f80-4498-11eb-98ca-0c40bfca51c2.png)

**Comment by jeremysanders** [comment]
Yes. If anyone wants to volunteer to make the icons more visible, that would be great. It just needs some messing with inkscape. The easiest trick is put a thicker, light-coloured copy of the icon underneath, to make them visible on a dark background.

**Comment by tjndr** [comment]
Is there any plans to add dark theme to macOS?

**Comment by jeremysanders** [comment]
I haven't got much time at the moment. Someone would probably need to bring this into the project: https://pypi.org/project/pyqt-darktheme/
